projects/instant-ui-library/src/lib/loading-spinner/loading-spinner/loading-spinner.component.html

```diff
@@ -0,0 +1,17 @@
+<div
+  id="spinner-{{ id }}"
+  [ngClass]="[
+    'spinner',
```

Comment:
A couple of ways to go about ngClass for reference.

```
[ngClass]="{
'spinner-small': sizeMode === size.SMALL,
'spinner-medium': sizeMode === size.MEDIUM
}"
```
Or 
```
[ngClass.spinner-small]="sizeMode === size.SMALL"
[ngClass.spinner-medium]="sizeMode === size.MEDIUM"
```
No more need for if else short hand. I prefer the first on.

Reply:
Noooooooice

Reply:
Refactored the button as well.